Add Spanish translation

diegombeltran opened the pull request:
Initial release. Colonies and Venux Next are missing, but we're working on adding them soon. Some strings could be missing, it's still work in progress. We'd like to have feedback from the Spanish players so we can find them sooner.

Spanish flag image is also missing (Language selector at start menu)

alrusdi commented on src/locales/cn/cards.json:
```diff
@@ -120,7 +120,6 @@
-    "1 VP PER 2 MICROBES ON THIS CARD.": "这张卡上每有2个微生物资源,获得1分",
     "Requires 4% oxygen. 1 VP per 2 Microbes on this card.": "需要至少4%的氧气浓度。这张卡上每有2个微生物资源,获得1分",
 
     "Release of Inert Gases": "释放惰性气体",
```
Spanish translation shouldn't contain any other translations. Please use empty string as placeholder

vsrisuknimit commented:
> Spanish flag image is also missing (Language selector at start menu)

Check out: assets/flags_responsive.png. Not sure if this is the correct flag. But you can add.

```
.language-icon--es {
    background-position: 2px 26.86%;
}
```

diegombeltran commented:
Hi, thank you for your feedback. I just bought Venux Next and Colonies, so we will translate all those strings and do as @alrusdi said.

We will also look into the css to set the flag up.

diegombeltran commented:
Hello. I added Venus/Colonies, new promo cards and refactored strings a little.
I have two questions:
- I tried to leave "" as a placeholder for not-yet-finished strings (Mostly Ares, Moon and some promo cards) Not sure if I understood right but does leaving and empty string as a placeholder fallback to English?
- The Spanish flag looks a bit odd with that emblem. I personally don't care but it may be more accurate removing it to avoid confusion.

alrusdi commented:
>     * I tried to leave "" as a placeholder for not-yet-finished strings (Mostly Ares, Moon and some promo cards) Not sure if I understood right but does leaving and empty string as a placeholder fallback to English?
> 

Yes, empty string is ignored in all translations 

>     * The Spanish flag looks a bit odd with that emblem. I personally don't care but it may be more accurate removing it to avoid confusion.

Do you have a demo server to look at it?

diegombeltran commented:
> > ```
> > * I tried to leave "" as a placeholder for not-yet-finished strings (Mostly Ares, Moon and some promo cards) Not sure if I understood right but does leaving and empty string as a placeholder fallback to English?
> > ```
> 
> Yes, empty string is ignored in all translations
> 
> > ```
> > * The Spanish flag looks a bit odd with that emblem. I personally don't care but it may be more accurate removing it to avoid confusion.
> > ```
> 
> Do you have a demo server to look at it?

I sent you a PM

diegombeltran commented:
> > ```
> > * I tried to leave "" as a placeholder for not-yet-finished strings (Mostly Ares, Moon and some promo cards) Not sure if I understood right but does leaving and empty string as a placeholder fallback to English?
> > ```
> 
> Yes, empty string is ignored in all translations
> 
> > ```
> > * The Spanish flag looks a bit odd with that emblem. I personally don't care but it may be more accurate removing it to avoid confusion.
> > ```
> 
> Do you have a demo server to look at it?

I do, but I rather not share it publicly. Can I send you an email?

bafolts commented:
@alrusdi I don't want to merge this if you are waiting for something. It seems good to merge to me. Are you waiting for any change?

alrusdi commented:
> @alrusdi I don't want to merge this if you are waiting for something. It seems good to merge to me. Are you waiting for any change?

No, I don't wait anything. Let's merge it!

diegombeltran commented:
Hi, @alrusdi , what is your Discord nickname? Thanks!

alrusdi commented:
> Hi, @alrusdi , what is your Discord nickname? Thanks!

the same as here - **alrusdi**
